client/src/api/review.ts

```diff
@@ -1,8 +1,31 @@
 import { AddReviewType } from '../models/AddReview';
+import ReviewViewModel from '../models/ReviewViewModel';
 import { UpdateReviewType } from '../models/UpdateReview';
+import { paginatedFetch, handleBadResponse } from '../utils/utils';
+import { PaginationRequest } from '../utils/constants';
+import { paginatedResponse } from '../utils/zodSchemas';
 import { getAuthToken } from './auth';
 
 const base = import.meta.env.VITE_API_BASE_URL;
+const ReviewSearchResponse = paginatedResponse(ReviewViewModel);
+
+export async function getReviews(bookId: string, body: PaginationRequest) {
+    const response = await paginatedFetch(base + '/Books/' + bookId + '/Reviews/search', body);
+    const data = await response.json();
+
+    if (!response.ok) {
+        await handleBadResponse(response);
+      }
+    return ReviewSearchResponse.parse(data);
+}
+
+/*
```

Comment:
jak zostawiamy jakiś zakomentowany kod to można dodawać po co zostaje bo potem niewiadomo albo się zapomina xd

Reply:
to jest do opinii, których pobieranie teraz chyba nie działa nie wiem, w każdym razie trzeba je przerobić